docs/docs/faq/question_07.md

```diff
@@ -0,0 +1,24 @@
+
+# Can I use a custom prior with sbi?
```

Comment:
this change exists twice now. It should not be part of this PR

Reply:
I did a whoopsie by branching of the other branch for this issue. Will rebase.